dev-docs/source/Workbench/RunningWorkbench.rst

```diff
@@ -0,0 +1,34 @@
+.. _RunningWorkbench:
```

Comment:
Could we link to this document from the PyCharm, VSCode & CLion pages? The flag is mentioned there I think but without explanation.